src/main/java/moviescraper/doctord/model/dataitem/MovieDataItem.java

```diff
@@ -39,7 +39,7 @@ public String dataItemSourceToString() {
 
 	public boolean isStringValueEmpty() {
 		String toStringValue = this.toString();
-		if (toStringValue.contains("=\"\""))
+		if (toStringValue.contains("=\"\" "))
```

Comment:
Need to add the extra space otherwise Titles that start with " (double quotes) is considered empty.